@here/harp-mapview/lib/MapView.ts

```diff
@@ -608,6 +608,13 @@ export interface MapViewOptions extends TextElementsRendererOptions {
      * @default 0
      */
     tilt?: number;
+
+    /**
+     * Set true to enable rendering mixed levels of detail (increases rendering performance).
+     *
+     * @default true
```

Comment:
The default should be `false` for now and maybe only enabled for the examples that use globe.

Reply:
My opinion would be to keep it true, it increases load performance, and have an example for the case where it should be disabled.